tests/samples/yaml/epics.ibek.yaml

```
@@ -0,0 +1,31 @@
+# yaml-language-server: $schema=file://schemas/ibek.schema.json
+module: epics
+definitions:
+  - name: EPICS_CA_MAX_ARRAY_BYTES
+    args:
+      - type: int
+        name: max_bytes
+        description: Max size in bytes for sending arrays over channel access
+        default: 6000000
+    env_vars:
+      - epicsEnvSet "EPICS_CA_MAX_ARRAY_BYTES", '{{ max_bytes }}'
```

I'd prefer if we didn't have `epicsEnvSet` in there, how about we change from `List[str]` to `Dict[str, str]`:
```suggestion
      EPICS_CA_MAX_ARRAY_BYTES: {{ max_bytes }}
```
Although then we should also change `define_args` to a `Dict[str, str]` to match

---

That does look tidier, I'll take a look